lib/ansible/modules/identity/keycloak/keycloak_client.py

```diff
@@ -39,57 +39,67 @@
 options:
     state:
         description:
-            - State of the client
+            - State of the client.
             - On C(present), the client will be created (or updated if it exists already).
-            - On C(absent), the client will be removed if it exists
-        choices: ['present', 'absent']
-        default: 'present'
+            - On C(absent), the client will be removed if it exists.
+        type: str
+        choices: [ absent, present ]
+        default: present
 
     realm:
         description:
             - The realm to create the client in.
+        type: str
+        default: master
 
     client_id:
         description:
-            - Client id of client to be worked on. This is usually an alphanumeric name chosen by
-              you. Either this or I(id) is required. If you specify both, I(id) takes precedence.
-              This is 'clientId' in the Keycloak REST API.
-        aliases:
-            - clientId
+            - Client id of client to be worked on.
+            - This is usually an alphanumeric name chosen by you.
+            - Either this or I(id) is required.
+            - If you specify both, I(id) takes precedence.
+            - This is C(clientId) in the Keycloak REST API.
+        type: str
+        aliases: [ clientId ]
 
     id:
         description:
-            - Id of client to be worked on. This is usually an UUID. Either this or I(client_id)
-              is required. If you specify both, this takes precedence.
+            - Id of client to be worked on.
+            - This is usually an UUID.
+            - Either this or I(client_id) is required.
+            - If you specify both, this takes precedence.
+        type: str
 
     name:
         description:
-            - Name of the client (this is not the same as I(client_id))
+            - Name of the client (this is not the same as I(client_id)).
+        type: str
 
     description:
         description:
-            - Description of the client in Keycloak
+            - Description of the client in Keycloak.
+        type: str
 
     root_url:
         description:
-            - Root URL appended to relative URLs for this client
-              This is 'rootUrl' in the Keycloak REST API.
-        aliases:
-            - rootUrl
+            - Root URL appended to relative URLs for this client.
+            - This is C(rootUrl) in the Keycloak REST API.
```

Comment:
Is C() appropriate in this case? I had struggled to find the correct quoting for this as it does not specifically relate to ansible options or values thereof, but rather a reference to what the option is called in the REST API (which could potentially be different from what the module refers to, though right now every such option should have an equally-named alias).

Reply:
It makes it stand out as a function/variable, so yes I think it makes sense.

Reply:
> right now every such option should have an equally-named alias

I have to say, I like that.